src/SFA.DAS.FindEmploymentSchemes.Contentful/GdsHtmlRenderers/GdsEmbeddedYoutubeContentRenderer.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Contentful.Core.Models;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.FindEmploymentSchemes.Contentful.GdsHtmlRenderers
+{
+    /// <summary>
+    /// A renderer for an embedded YouTube video
+    /// </summary>
+    public class GdsEmbeddedYoutubeContentRenderer : IContentRenderer
```

Comment:
Do these renderers not have unit tests?

Reply:
The renderers have only been used up to now, as part of an offline utility, run to generate C# code.

We'll definitely need unit tests for them as part of the Contenful integration (when they will run online), which is currently being developed.

As part of that development, we've updated the renderers to support the latest version of the Contentful .Net library, which has changed their signatures, so it makes sense for the unit tests to be written as part of that story.